src/Common/Auth/OpenIDConnect/Entities/UserEntity.php

```diff
@@ -119,29 +119,38 @@ public function getUserAccount($userId)
 
     protected function getAccountByPassword($username, $password, $email = ''): ?bool
     {
+        // first check core openemr authentication
         $auth = new AuthUtils('api');
```

Comment:
There's nothing wrong with the way I had it.
The auth class will tell us who the username/password pair belong to without individually testing each role.
I did this on purpose in case we add new roles or decide the patient auth wasn't tied strictly to portal.
I've tested this a hundred times so how is it broke?